docs/source/usersguide/processing.rst

```diff
@@ -97,10 +97,11 @@ VTK Mesh File Generation
 ------------------------
 
 VTK files of OpenMC meshes can be created using the
-:meth:`openmc.Mesh.write_data_to_vtk`  method. Data can be applied to the elements
-of the resulting mesh from mesh filter objects. This data can be provided either
-as a flat array or, in the case of structured meshes (i.e. ``Regular``,
-``Rectilinear``, ``Cylindrical``, and ``Spherical`` meshes), the data can be
+:meth:`openmc.Mesh.write_data_to_vtk`  method. Data can be applied to the
+elements of the resulting mesh from mesh filter objects. This data can be
+provided either as a flat array or, in the case of structured meshes
+(:class:`~openmc.RegularMesh`, :class:`~openmc.RectilinearMesh`,
+:class:`~openmc.CylindricalMesh`, or :class:`SphericalMesh`), the data can be
```

Comment:
```suggestion
(:class:`~openmc.RegularMesh`, :class:`~openmc.RectilinearMesh`,
:class:`~openmc.CylindricalMesh`, or :class:`~openmc.SphericalMesh`), the data can be
```

Also @pshriwise what does the ~ sign do in this case?

Reply:
This allows the class to render in the documentation without the full module path.

https://www.sphinx-doc.org/en/master/usage/referencing.html